core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala

```diff
@@ -205,6 +205,8 @@ object BrokerApiVersionsCommand {
   private object AdminClient {
     val DefaultConnectionMaxIdleMs = 9 * 60 * 1000
     val DefaultRequestTimeoutMs = 5000
+    val DefaultSocketConnectionSetupMs = 10 * 1000
+    val DefaultSocketConnectionSetupMaxMs = 127 * 1000
```

Comment:
It seems that we can remove these two and use the ones in `CommonClientConfigs`.

Reply:
Yes. Removed.

Reply:
Looks like the constants haven't been removed.

Reply:
Oh, that's true. Removed them.